SmartDeviceLink/SDLRequestType.h

```diff
@@ -105,6 +105,11 @@ extern SDLRequestType const SDLRequestTypeEmergency;
 extern SDLRequestType const SDLRequestTypeMedia;
 
 /**
- A firmeware over-the-air request
+ A firmware over-the-air request
  */
 extern SDLRequestType const SDLRequestTypeFOTA;
+
+/**
+ A request that is OEM specific using the `RequestSubType` in SystemRequest
+ */
+extern SDLRequestType const SDLRequestTypeOEMSpecific;
```

Comment:
The new enum case is missing from the test cases in _SDLRequestTypeSpec_

Reply:
Fixed